tests/3_claim.ts

```diff
@@ -50,17 +51,28 @@ async function main2() {
     claimContents,
     claimer.address
   )
+
   /* 🚧 2️⃣ COPY_START for requestForAttestation_example (below this comment) 🚧 */
-  const {
-    message: requestForAttestation,
-  } = await Kilt.RequestForAttestation.fromClaimAndIdentity(claim, claimer)
+  const requestForAttestation = Kilt.RequestForAttestation.fromClaimAndIdentity(
+    claim,
+    claimer
+  )
 
   // log this so you can paste it locally
   console.log(
-    'requestForAttestationJSONString: ',
+    'requestForAttestationJSONString:\n',
     JSON.stringify(requestForAttestation)
   )
   /* 🚧 2️⃣ COPY_END for requestForAttestation_example (above this comment) 🚧 */
 }
+// execute calls
+
+async function execution() {
+  await setup()
+  main()
+  main2()
```

Comment:
I guess it's fine if these run at the same time?

Reply:
Yep its fine :)